src/components/List.js

```diff
@@ -125,6 +128,13 @@ const List = ({ token }) => {
     if (daysUntilNextPurchase >= 30) return 'not-too-soon';
   };
 
+  const checkboxColor = {
+    inactive: 'lightgrey',
```

Comment:
I initially had the object with all keys in single quotes, not sure when this happened, but now I'm thinking do quotes matter for object keys? 🤔

Reply:
Hmm... it looks like it's working either way. I'm not sure which way is best-practice. May want to just pick one style and make them all the same? I feel like I've usually seen object keys without quotes, but prob good to get more opinions on this.